src/Components/AddProject.js

```diff
@@ -0,0 +1,55 @@
+import React, { Component } from 'react';
+
+class AddProject extends Component {
+  constructor(){
+      super();
+      this.state = {
+        newProject: {}
+      }
```

Comment:
Can you please elaborate need for this `state `here

Reply:
You can achieve this using a simple object variable.

Reply:
Please provide some reference.